test/controllers/donation_goal_controller_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule CodeCorps.DonationGoalControllerTest do
+  use CodeCorps.ApiCase
+
+  alias CodeCorps.DonationGoal
+
+  @valid_attrs %{title: "A donation", description: "A description", amount: 200, current: false}
+  @invalid_attrs %{title: nil, description: nil}
+  @resource_name :donation_goal
+
+  # TODO: This could go into a common module
```

Comment:
I was going to say, these all look like they could be extracted out into some helpers.